src/server/db_slice.h

```diff
@@ -544,6 +556,7 @@ class DbSlice {
   // Used in temporary computations in Acquire/Release.
   mutable absl::flat_hash_set<uint64_t> uniq_fps_;
 
+  mutable util::fb2::SharedMutex cb_mu_;
```

Comment:
Please add this comment, you can remove the comments above LockChangeCb/UnlockChangeCb
To ensure correct data replication, we must serialize the buckets that each running command will modify, followed by serializing the command to the journal. We use a mutex to prevent interleaving between bucket and journal registrations, and the command execution with its journaling. LockChangeCb is called before the callback, and UnlockChangeCb is called after journaling is completed. Register to bucket and journal changes is also does without preemption